package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "lint:prettier": "prettier --check _posts _drafts",
+    "lint:typos": "typos _posts _drafts",
+    "lint:prettier:fix": "prettier --write  _posts _drafts"
+  },
+  "dependencies": {
+    "lefthook": "^1.6.16",
+    "prettier": "^3.3.2"
+  },
+  "devDependencies": {
+    "typos-rs-npm": "dalisoft/typos-rs-npm"
```

Comment:
Suggestion - you could add a empty `_typos.toml` file to anticipate us needing to add false positives (I just ran it on my LTE article and UE wasn't recognized as a valid word). I'm good with adding it when we need it though!

Reply:
good call, done!